deps/uv/include/uv.h

```diff
@@ -138,6 +138,7 @@ extern "C" {
   XX(EOF, "end of file")                                                      \
   XX(ENXIO, "no such device or address")                                      \
   XX(EMLINK, "too many links")                                                \
+  XX(EHOSTDOWN, "host is down")                                               \
```

Comment:
That seems like another API change that might be a bit more difficult not to use within the node code base, but at that point I don't know what other options we have.